README.md

```diff
@@ -206,7 +221,8 @@ To run an EventQL query, call the `runEventQlQuery` function and provide the que
 
 ```php
 $rows = $client->runEventQlQuery(
-  'FROM e IN events PROJECT INTO e',
+  'FROM e IN events' .
+  'PROJECT INTO e'
```

Comment:
Hier fehlt in der Query eine Leerstelle zwischen `events` und `PROJECT`.

Wäre es nicht besser, das als Einzeiler zu machen?